pkg/resolution/resolver/cluster/resolver.go

```diff
@@ -161,9 +182,17 @@ func (r *Resolver) isDisabled(ctx context.Context) bool {
 // ResolvedClusterResource implements framework.ResolvedResource and returns
 // the resolved file []byte data and an annotation map for any metadata.
 type ResolvedClusterResource struct {
-	Content   []byte
-	Name      string
+	// Content is the actual resolved resource data.
+	Content []byte
+	// Spec is the data in the resolved task/pipeline CRD spec.
+	Spec []byte
+	// Name is the resolved resource name in the cluster
+	Name string
+	// Namespace is the namespace in the cluster under which the resolved resource was created.
 	Namespace string
+	// ResourceURI is in the format of namespace-scoped resources API.
+	// https://kubernetes.io/docs/reference/using-api/api-concepts/#resource-uris
```

Comment:
nit: This isn't quite a resource URI (it's `<resource uri>@<uid>`). We should update the comment and/or name to reflect this.

Reply:
Good catch. Updated both comment and name, and also updated the clusterresolver doc. Thank you!